lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -5940,6 +5942,14 @@ struct AtomicRMWOpConversion
 
     auto vecTy = vec_ty(valueElemTy, vec);
     auto elemsPerThread = getElemsPerThread(val.getType());
+    // mask
+    Value mask = int_val(1, 1);
+    auto shape = valueTy.getShape();
```

Comment:
In addition to test python E2E test, do we need to add some more low-level test for the mask operations?

Reply:
The E2E test uses implicit mask.  Low-level mask means explicit mask? I can add after scalar support.